lib/process-services-cloud/src/lib/process/process-filters/components/edit-process-filter-cloud.component.spec.ts

```diff
@@ -116,6 +117,42 @@ describe('EditProcessFilterCloudComponent', () => {
         });
     });
 
+    it('should not display mat-spinner if isloading set to false', () => {
```

Comment:
If you use whenStable you need to use the done or async is the same across all the unit test in this file